### PR TITLE
Date picker and File browser

### PR DESCRIPTION
**Pull request by DavidBachmann**
screen shot 2017-03-06 at 3 09 13 pm

DatePicker and file browser

image

### CODE REVIEW

File: packages/tux/src/components/fields/DatePicker.tsx [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+import classNames from 'classnames'
+import moment from 'moment'
+import DayPicker, { DateUtils } from 'react-day-picker'
+import { tuxColors, tuxInputStyles } from '../../styles'
+import { tuxDatePickerStyles } from './DatePicker.styles'
+import { fade } from '../../utils/color'
+
+const DATE_FORMAT = 'YYYY-MM-DD'
+
+export interface State {
```
Comment (baering) [inline packages/tux/src/components/fields/DatePicker.tsx:11]: I don't think you need to export this

**Comment by thorsteinsson** [comment]
🎉

**Comment by baering** [comment]
➕ 🍾

File: packages/tux/src/components/fields/DatePicker.tsx (hunk 1 above)
Comment (DavidBachmann) [reply packages/tux/src/components/fields/DatePicker.tsx:11]: Without export
image

With Export
image

**Pull request merged**